kuma/javascript/src/l10n.test.js

```diff
@@ -2,6 +2,12 @@
 
 import { localize, getLocale, gettext, ngettext, interpolate } from './l10n.js';
 
+// We don't actually want the strings in this file to be extracted
+// for localization, so we're going to use non-standard aliases
+// for gettext and ngettext.
+const gtext = gettext;
+const ngtext = ngettext;
```

Comment:
Couldn't you have done `import { gettext as original_gettext } from './l10n.js'` so that the rest of the code gets the familiar looking function name.

Reply:
Oh wait. Silly me. I suspect the Duente (or whatever it's called) isn't *that* smart.